node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -372,7 +382,7 @@ class NodeRuntime private[node] (
       _     <- info
       local <- peerNodeAsk.ask
       host  = local.endpoint.host
-      servers <- acquireServers(apiServers, reportingCasper)(
+      servers <- acquireServers(apiServers, reportingCasper, webApi)(
```

Comment:
could add startup message here for web API? https://github.com/rchain/rchain/blob/af4b878c20dc64e2c5333ad25b57179d2ac5ead2/node/src/main/scala/coop/rchain/node/NodeRuntime.scala#L387